docs/source/index.rst

```diff
@@ -0,0 +1,291 @@
+cluster_generator
+=================
+
+|yt-project| |docs| |testing| |Github Page| |Pylint| |coverage| |ncodes|
+
+.. raw:: html
+
+   <hr style="height:10px;background-color:black">
+
+The `Cluster Generator Project <https://eliza-diggins.github.io/cluster_generator>`_ (CGP) is a cross-platform Python library for generating initial conditions of galaxy clusters for N-body / hydrodynamics codes.
+CGP provides a variety of construction approaches, different physical assumption, profiles, and gravitational theories. Furthermore, the CGP is intented to interface with
+the vast majority of N-body / hydrodynamics codes, reducing the headache of converting initial conditions between formats for different simulation softwares. GCP's goal is to provide
+comprehensive tools for modeling and implementation of galaxy clusters in astrophysical simulations to promote the study of galaxy cluster dynamics.
+
+This repository contains the core package, which is constructed modularly to facilitate easy development by users to meet particular scientific use cases. All of the
+necessary tools to get started building initial conditions are provided.
+
+You can access the documentation `here <http://eliza-diggins.github.io/cluster_generator>`_, or build it from scratch using the ``./docs`` directory in this source distribution.
+
+Developement occurs here on Github, if you encounter any bugs, issues, documentation failures, or want to suggest features, we recommend that you submit an issue on
```

Comment:
Developement --> Development

Reply:
@jzuhone Fixed in most recent commit. Also added a spell checker to the pre-commit hooks for the next commit to help filter some of these out! I tend to make a lot of typos!